packages/ember-simple-auth-devise/README.md

```diff
@@ -57,17 +57,16 @@ needed the format handling can be left out of course_):
 
 ```ruby
 class SessionsController < Devise::SessionsController
+  respond_to :html, :json
+
   def create
-    respond_to do |format|
-      format.html { super }
-      format.json do
-        self.resource = warden.authenticate!(auth_options)
-        sign_in(resource_name, resource)
+    super do |user|
+      if request.format.json?
         data = {
-          token:      self.resource.authentication_token,
-          user_email: self.resource.email
+          user_token: user.authentication_token,
```

Comment:
This was renamed to just `token` in Ember Simple Auth's last release as [`ActionController::HttpAuthentication::Token`](http://api.rubyonrails.org/classes/ActionController/HttpAuthentication/Token.html) expects the token to be sent as `Authorization: Token token=…` and we wanted to have a consistent name for the token property that's used throughout the entire library.